frame/system/src/lib.rs

```diff
@@ -533,6 +561,11 @@ decl_module! {
 		}
 
 		/// Set some items of storage.
+		/// 
+		/// # <weight>
+		/// - `O(VI)` where `V` length of `items` and `I` length of one item
+		/// - `V` storage writes (codec `O(I)`).
```

Comment:
Shouldn't we consider that a storage write is constant complexity? https://github.com/paritytech/substrate/issues/2966#issuecomment-510087309

So complexity would be `O(V)` and so with benchmark we can find `w = K1 + K2*V` otherwise how can we find solution with benchmark ? if we do `w = K1 + K2*V*I` then I'm not we can find any value for K2 because I think `w(V, I)` is somewhat equal to `w(V, I*2)`.

Reply:
The thing is that there is no bound (that I could see by looking at the code here) on the size of the item. AFAICS it's just a vec and could thus be arbitrarily big.
Do let me know if you know of some bounds for the item size, though.

Reply:
If we can find a correct upper bound for the size of value where it is still kind of constant complexity we can benchmark agains't this upper bound and say in the doc of this extrinsic that if `I` is more than this upper bound then weight can be off.

(note that such documentation constrain is only possible because this is root exstrinsic)